llvm/lib/Transforms/PeepholeOptimizationCourse/ObfuscateI8AddNewPassManager/ObfuscateI8AddNewPassManager.cpp

```diff
@@ -0,0 +1,77 @@
+#include "llvm/Transforms/PeepholeOptimizationCourse/ObfuscateI8AddNewPassManager.h"
+
+#include "llvm/Pass.h"
+
+#include "llvm/IR/Function.h"
+#include "llvm/IR/InstIterator.h"
+#include "llvm/IR/Constants.h"
+#include "llvm/IR/LegacyPassManager.h"
+#include "llvm/IR/IRBuilder.h"
+#include "llvm/Transforms/Utils/BasicBlockUtils.h"
+
+
+#include "llvm/Support/raw_ostream.h"
+
+#include "llvm/Transforms/IPO/PassManagerBuilder.h"
+
+using namespace llvm;
+
+bool isBinaryAdd(Instruction const* instruction) {
```

Comment:
Make all free functions in this source file of internal linkage

Reply:
Did it in b5d1c509303a012d1cb5615a407b29484c940a3b